tests/safeds/data/tabular/transformation/test_standard_scaler.py

```diff
@@ -0,0 +1,241 @@
+                    {
+                        "col1": [3, 3],
+                    },
+                ),
+            ),
+        ],
+    )
+    def test_should_return_transformed_tables(
+        self,
+        fit_and_transform_table: Table,
+        only_transform_table: Table,
+        column_names: list[str] | None,
+        expected_1: Table,
+        expected_2: Table,
+    ) -> None:
+        s = StandardScaler().fit(fit_and_transform_table)
+        assert s.fit_and_transform(fit_and_transform_table, column_names) == expected_1
+        assert s.fit_and_transform(only_transform_table, column_names) == expected_2
+
+
+class TestFitAndTransform:
+    def test_should_not_change_original_table(self) -> None:
+        table = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        StandardScaler().fit_and_transform(table)
+
+        expected = Table(
+            {
+                "col1": [0.0, 5.0, 10.0],
+            },
+        )
+
+        assert table == expected
+
+    def test_get_names_of_added_columns(self) -> None:
+        transformer = StandardScaler()
+        with pytest.warns(
+            UserWarning,
+            match="StandardScaler only changes data within columns, but does not add any columns.",
+        ), pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_added_columns()
+
+        table = Table(
+            {
+                "a": [0.0],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        with pytest.warns(
+            UserWarning,
+            match="StandardScaler only changes data within columns, but does not add any columns.",
+        ):
+            assert transformer.get_names_of_added_columns() == []
+
+    def test_get_names_of_changed_columns(self) -> None:
+        transformer = StandardScaler()
+        with pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_changed_columns()
+        table = Table(
+            {
+                "a": [0.0],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        assert transformer.get_names_of_changed_columns() == ["a"]
+
+    def test_get_names_of_removed_columns(self) -> None:
+        transformer = StandardScaler()
+        with pytest.warns(
+            UserWarning,
+            match="StandardScaler only changes data within columns, but does not remove any columns.",
+        ), pytest.raises(TransformerNotFittedError):
+            transformer.get_names_of_removed_columns()
+
+        table = Table(
+            {
+                "a": [0.0],
+            },
+        )
+        transformer = transformer.fit(table, None)
+        with pytest.warns(
+            UserWarning,
+            match="StandardScaler only changes data within columns, but does not remove any columns.",
+        ):
+            assert transformer.get_names_of_removed_columns() == []
+
+
+class TestInverseTransform:
```

Comment:
Nice, you could as well add a test method that should handle edge cases such as handling of passed empty tables or empty columns.